lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php

```diff
@@ -319,6 +328,26 @@ public function getMigrationsColumnName()
     }
 
     /**
+     * Set the length of the migration string column.
+     *
+     * @param int $columnLength The migration string column length.
+     */
+    public function setMigrationsColumnLength($columnLength)
```

Comment:
The codebase I'm PR'ing against doesn't have type declarations, so I was trying to stay consistent with that.